sqdrift/sqd.ipynb

```
@@ -0,0 +1,566 @@
+    "\n",
+    "# Minimal set van 10 termen voor qDRIFT (1-body Z-termen op qubits 0..9)\n",
+    "selected_fermion_terms_10 = [{\"type\": \"Z\", \"qubits\": (q,), \"coeff\": 1.0} for q in range(10)]\n",
+    "\n",
+    "qc = build_sqdrift_circuit(\n",
+    "    n_qubits=48,\n",
+    "    excitations=selected_fermion_terms_10,  # 10 per circuit voor ondiepe batch\n",
+    "    coeffs=np.abs([t[\"coeff\"] for t in selected_fermion_terms_10]),\n",
+    "    t=1.0, k=1, N_steps=50\n",
+    ")\n",
+    "\n",
+    "qc_t = transpile(\n",
```

Ruff (0.12.2)

Redefinition of unused `sort_and_remove_duplicates` from line 115

Remove definition: `sort_and_remove_duplicates`

(F811)